src/server/api/schema.js

```diff
@@ -849,11 +849,6 @@ const rootMutations = {
         })
         .returning("*");
 
-      const memoizer = await MemoizeHelper.getMemoizer();
-      await memoizer.invalidate(cacheOpts.CampaignsList.key, {
-        organizationId: organization.id
-      });
```

Comment:
We're doing it inside `editCampaign` instead now! https://github.com/politics-rewired/Spoke/blob/6088c8a41b324ac48c0d28beb251309065a030e4/src/server/api/lib/campaign.ts#L736

This should fix the invalidate happening when something happens, and then someone else possibly calling the campaigns list, and us getting back stale campaigns, instead the invalidate now happens at the end of `editCampaign` since we do a `return editCampaign` in both scenarions (this and below).